src/components/approval-type/ApprovalType.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div id="approval-type">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label id="approval-type-label">Approval Type</label>
+      </v-col>
+      <v-col cols="12" sm="9">
+        <v-radio-group class="payment-group" v-model="approvalTypeSelected" @change="radioButtonChanged">
+          <!-- COURT ORDER radio button -->
+          <v-radio id="court-order-radio" class="mb-0"
+             :label="getRadioText(ApprovalTypes.VIA_COURT_ORDER)"
+             :value="ApprovalTypes.VIA_COURT_ORDER"
+          />
+          <v-form ref="courtNumRef" id="court-num-form" v-model="valid" class="mt-4 ml-8">
+            <v-text-field
+              @input="courtOrderNumberChanged"
+              id="court-order-number-input"
+              v-model="courtOrderNumber"
+              label="Court Order Number"
+              :rules="courtOrderNumRules"
+              :disabled="approvalTypeSelected === ApprovalTypes.VIA_REGISTRAR"
+              @update:error="emitValidationError($event)"
+              filled
+            />
```

Comment:
Once again, super nitpicky, we try to lead with the id, followed by properties/bound properties and subsequently event handlers. 
So in this case, maybe just move the `input` event to bottom with `update` etc. 
Not required at all, just general practice.

Reply:
Thanks for letting me know.  I'll make the changes you suggest.